dts/arm/st/l4/stm32l471.dtsi

```diff
@@ -188,5 +188,13 @@
 				#pwm-cells = <2>;
 			};
 		};
+
+		sdmmc1: sdmmc1@40012800 {
+			compatible = "st,stm32-sdmmc";
+			reg = <0x40012800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000400>;
+			status = "disabled";
+			label = "SDMMC1";
```

Comment:
Minor detail: It seems everywhere else there is an underscore before the 1. So `SDMMC1` -> `SDMMC_1`.

Reply:
Fixed, thanks.